CMakeLists.txt

```diff
@@ -156,15 +156,52 @@ if (EXISTS .gitignore)
 
   set(CPACK_SOURCE_GENERATOR ZIP)
   set(CPACK_SOURCE_IGNORE_FILES ${ignored_files})
-  set(CPACK_SOURCE_PACKAGE_FILE_NAME cppformat-${CPPFORMAT_VERSION})
```

Comment:
Why remove `CPACK_SOURCE_PACKAGE_FILE_NAME`? It is used to set the correct source package name for distribution via website and releases.